test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -242,6 +241,19 @@ private static void checkEquals(long l1, long l2) throws Exception {
     }
   }
 
+  /**
+   * Logs msg at debug level if inTestingContext, info level otherwise
+   *
+   * @param msg message to be logged
+   */
+  private static void logMessage(String msg) {
```

Comment:
Assumed (based on previous comments) that the message should be logged in debug mode if we are in the testing context, and info mode otherwise. Should this always be log.info?

Reply:
My reading is that is one possibility.  Depending on the frequency of some of the messages, maybe some are debug and others are at info?

Then using log4j configuration can be tuned for the circumstance.

Reply:
Changing everything to info should be fine.  The test are already configured to log debug output [here](https://github.com/apache/accumulo/blob/main/test/src/main/resources/log4j2-test.properties), so changing to info would not increase the amount output by the test when running it from maven.